graphql-schemas/schema.yml

```diff
@@ -879,7 +937,7 @@
       alb: NamespaceTerraformResourceALB_v1
       secrets-manager: NamespaceTerraformResourceSecretsManager_v1
       asg: NamespaceTerraformResourceASG_v1
-      route53-zone: NamespaceTerraformResourceRoute53Zone_v1
+      dns-zone: NamespaceTerraformResourceRoute53Zone_v1
   fields:
   - { name: provider, type: string, isRequired: true }
   - { name: account, type: string, isRequired: true }
```

Comment:
why is the account still mandatory if it is practically the provisioner one level up?

Reply:
added a comment about backwards compatibility